mita/controllers/nodes.py

```diff
@@ -168,9 +168,11 @@ def index(self):
             # that means that they are probably busy, so create a new one
             for n in matching_nodes:
                 difference = now - n.created
-                if difference.seconds < 480:  # 8 minutes
+                if difference.seconds < 240:  # 4 minutes
```

Comment:
I feel like this will almost guarantee an extra node created for each job. Maybe we should start with 6 minutes instead?